src/services/notification.service.ts

```diff
@@ -146,6 +146,55 @@ export class NotificationService {
     return;
   }
 
+  async sendVoteCount(
+    referencetype: ReferenceType,
+    referenceID: string,
+  ): Promise<void> {
+    const destination: Promise<string> = new Promise(resolve => {
+      if (referencetype === ReferenceType.POST) {
+        this.postRepository
+          .findById(referenceID, {
+            include: [{relation: 'User'}],
+            fields: ['createdBy'],
+          })
+          .then(result => {
+            return result.createdBy;
+          })
+          .catch((err: Error) => {
+            throw err;
+          });
+      } else {
+        this.commentRepository
+          .findById(referenceID, {
+            include: [{relation: 'User'}],
```

Comment:
what is the purpose of include? I think we can remove it, since we only need userId